### DISCUSSION TITLE
Text color opacity for disabled state

### DISCUSSION THREAD

**Discussion by hamanuha** [tailwindlabs/tailwindcss#9696] [Help] [answered]
Hi Tailwind team,

thanks for this great framework. I really enjoy using it.

I want to use a text color with opacity for the disabled state:
`disabled:text-cyan-700/60`

But it seems the opacity is just ignored.
Is this a bug or do I have to enable something in the config file?

**Comment by wongjn** [comment, marked as answer]
Nope, [should be working out-of-the-box as demonstrated here](https://play.tailwindcss.com/YQNm8qxvfi). Some things you could check:
- Check the class has been generated in your CSS. If it is not present, it could indicate a problem with your configuration, such as `content` file globs not sufficiently covering your source code.
- Check the element that you are adding the class to has the `disabled` attribute on it.
- Check that there is no other CSS rules that might be overriding the rule.

**Reply by hamanuha** [reply]
OMG, this was really stupid...
I used the same color as the background color and expected a lighter version of the background color.
But instead I should have used white with opacity as text color.
Everything working as expected.
